Model/Database/Contexts/MockDatabaseContext.cs

```diff
@@ -20,6 +20,8 @@ public class MockDatabaseContext : IDatabaseContext
         public override DbSet<Permission> Permissions { get; set; }
         public override DbSet<RolePermissions> RolePermissions { get; set; }
         public override DbSet<Artist> Artists { get; set; }
+        public override DbSet<Album> Albums { get; set; }

```

Comment:
Van deze en AlbumsArtistSongs moeten nog mock tabbel gemaakt. In de constructor van MockDatabaseContext.